gpiozero/boards.py

```diff
@@ -949,10 +949,20 @@ def __init__(self, *pins, dp=None, font=None, pwm=False, active_high=True,
 
         if font is None:
             if len(pins) == 7:
-                with resources.files('gpiozero.fonts').joinpath('7seg.txt').open() as f:
+                try:
+                    fontpath = resources.files('gpiozero.fonts').joinpath('7seg.txt')
```

Comment:
Seems to be a lot of repetition here (between 7seg and 14seg), so I wonder if it might be worth abstracting it out into a separate helper function?

Reply:
Yes, good idea.
I'd suggest I move it up to the imports at the top, to avoid having `import` hidden in the code and use:

```python
from importlib import resources
try:
    files = resources.files
except AttributeError:
    import importlib_resources
    files = importlib_resources.files
```

That would probably be the best compromise between not importing in the middle of code and not running code during the imports.(?)

Reply:
The only caveat I'd add is that `files` is perhaps too generic of a name to exist at global scope - maybe `resource_files` ?

Reply:
I think I found the answer: positioning just under the imports, named `fontslocation`and pointing to the path where the fonts reside...

Reply:
Looks a lot neater :+1: